perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
Not really. The methods we have in other classes are private methods that parse the input. Those methods are usually static methods so everyone can use them only to parse a stream of data. Specific methods that parse some parts of the streams are private and not static.
If you think the methods you have defined will be used by someone at some point, they should be public and static but as they are parsing specific (and this class doesn't have any parsing method) I think they should be out of this class.